extensions/odoo_theme/static/scss/_custom_alerts.scss

```diff
@@ -0,0 +1,99 @@
+$o-alert-states: (
+    "note": ("color": $note, "icon": $i-note),
+    "tip": ("color": $tip, "icon": $i-lightbulb),
+    "example": ("color": $example, "icon": $i-knowledge),
+    "exercise": ("color": $example, "icon": $i-exercise),
+    "go-to": ("color": $go-to, "icon": $i-arrow-right),
+    "warning": ("color": $warning, "icon": $i-warning),
+    "danger": ("color": $danger, "icon": $i-danger)
+);
+
+.alert {
+    $color: $o-blue;
+    $background: shift-color($color, $alert-bg-scale);
+    $border: shift-color($color, $alert-border-scale);
+    $color: shift-color($color, $alert-color-scale);
+    @if (contrast-ratio($background, $color) < $min-contrast-ratio) {
+        $color: mix($color, color-contrast($background), abs($alert-color-scale));
+    }
+    position: relative;
+    display: inline-block;
+    width: 100%;
+    padding: $alert-padding-y $alert-padding-x;
+    border: 0 solid transparent;
+    border-left-width: $alert-border-width;
+    border-radius: 0 $alert-border-radius $alert-border-radius 0;
+    @include font-size($font-size-secondary);
+    @include alert-variant($background, $border, $color);
+
+    .alert-title, > h3 {
+        line-height: 1em;
+        margin: 0 0 10px 0;
+        @include font-size($font-size-base);
+        font-weight: $fw_bold;
+        &:before {
+            content: $i-info;
+            font-family: 'icomoon' !important;
+            @include font-size($font-size-base);
+            font-weight: inherit;
+            vertical-align: baseline;
+            margin:  0 .2rem 0 0;
+        }
+    }
+    > p , > ul {
+        margin-bottom: .5rem;
+    }
+    > *:last-child {
+        margin-bottom: 0;
+    }
+    + * {
+        margin-top: $margin-s;
```

Comment:
I think you didn't get the point of this rule.
This rule is to remove the margin-bottom on all the direct last-children **inside** the alert card, not the alert card's margin-bottom.
Elements such as `p, .highlight-default, ul` have `margin-bottom`, which when used outside the alert are needed. Since the doc content is constantly changing and the content makers are capable of placing whatever they want inside the alerts, we may face a case where simply listing individual elements is not enough. But it is only a margin so I'll simply do that :)